app/controllers/katello/api/v2/distributions_controller.rb

```diff
@@ -1,38 +0,0 @@
-module Katello
-  class Api::V2::DistributionsController < Api::V2::ApiController
```

Comment:
Pretty much.  I'm sure others would disagree, but really this api doesn't even make sense.  the index call requires a repository_id and returns a list, even though there will ever be one. 

We could maintain this api (minus the searching) with these attributes:   :version, :arch, :family, :variant  simply by mapping the repo object i guess.  That would be fairly simple.  Maintaining the full api (minus search syntax compatibility, which i guess we've determined is okay to change) would basically dictate how the model is done (and not done the way we discussed).